Articles/_posts/Forum/What can you do with YFI/2021-05-05-What-can-you-do-with-YFI.md

```diff
@@ -4,322 +4,322 @@ title:  "What can you do with YFI?"
 
-#### Tools
+#### Alat-Alat
 
 - [Ceazor's Guide to Bancor v2.1: YFI/BNT Single-Sided Protected Pool](https://youtu.be/LhLMhizDNwE?t=23)
 - [Bancor Documentation](https://docs.bancor.network/)
 - [Bancor Dune dashboard](https://duneanalytics.com/Bancor/bancor_1)
 - [Bancor v2.1 Staking for (DeFi) Dummies](https://blog.bancor.network/bancor-v2-1-staking-for-defi-dummies-f104a6a8281e)
 - [Zap into YFI/BNT Single-Sided Protected Pool](https://zapper.fi/invest?protocol=bancor&contractAddress=0x555981b82743aa443cb1c3002c85ba9d795d4596&modal=legacy-invest)
 
-#### Risks
+#### Resiko
 
-Bancor smart contract risk and trusting the [bancor v2.1 model](https://blog.bancor.network/proposing-bancor-v2-1-single-sided-amm-with-elastic-bnt-supply-bcac9fe655b). Bancor is [audited](https://docs.bancor.network/ethereum-contracts/security).
+Resiko kontrak cerdas Bancor dan memercayai [bancor v2.1 model](https://blog.bancor.network/proposing-bancor-v2-1-single-sided-amm-with-elastic-bnt-supply-bcac9fe655b). Bancor di [audit](https://docs.bancor.network/ethereum-contracts/security).
 
-##### Insurance
+##### Asuransi
 
-For DeFi insurance, you would only need to insure Bancor to have full coverage in case of a smart contract bug that led to the loss of funds.  
+Untuk asuransi DeFi, Anda hanya perlu memastikan Bancor memiliki cakupan penuh jika terjadi bug kontrak pintar yang menyebabkan hilangnya dana.  
 
 #### APY
 
 APY: 3.7%
-Calculation: Last month, annualized.
-Source: Bancor Team
+Perhitungan: Bulan lalu, disetahunkan.
+Sumber: Bancor Team
 
-- [YFI Pool APY](https://duneanalytics.com/queries/30793/62118): Single click on each pool until only YFI is left. This will show APY stats for the pool.
+- [YFI Pool APY](https://duneanalytics.com/queries/30793/62118): Satu klik pada setiap pool sampai hanya YFI yang tersisa. Ini akan menunjukkan statistik APY untuk pool.
 
 ___
 
 ## ***[Collateralized Debt Position](https://defipulse.com/blog/what-is-a-cdp)***
 
-CDP-based protocols allow you to deposit your YFI in exchange for a lesser amount of the protocol-issued token, which is usually a stablecoin. You can then use the issued token to farm and generate yield. Using this method is more time-intensive than the others and requires constant monitoring of the CDP to make sure you're above the liquidation ratio. Also, it requires you to use the stablecoins minted to find yield elsewhere and manage that as well.
+Protokol berbasis CDP memungkinkan Anda untuk menyetor YFI Anda dengan imbalan token yang dikeluarkan protokol dalam jumlah yang lebih sedikit, yang biasanya berupa stablecoin. Anda kemudian dapat menggunakan token yang dikeluarkan untuk bertani dan mendapatkan hasil. Menggunakan metode ini lebih memakan waktu daripada yang lain dan membutuhkan pemantauan CDP yang konstan untuk memastikan Anda berada di atas rasio likuidasi. Juga, Anda harus menggunakan stablecoin yang dicetak untuk menemukan yield di tempat lain dan mengelolanya juga.
```

Comment:
what word should we choose for `yield`?

a. hasil
b. penghasilan
c. panen

wdyt @gateway-504